packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -56,12 +56,15 @@ typedef ExpansionPanelHeaderBuilder = Widget Function(BuildContext context, bool
 /// Expansion panels are only intended to be used as children for
 /// [ExpansionPanelList].
 ///
+/// See [ExpansionPanelList] for a sample implementation.
```

Comment:
It may be worth turning this into a template rather than a link here - although I'm not sure if you can do that with with sample code or not.  @gspencergoog do you know?

Reply:
@dnfield Could you clarify what you mean here by turning it into a template?

Reply:
He means add `{@template flutter.material.expansion_panel.listExample}`/`{@endtemplate}` around the example code, and putting `{@macro flutter.material.expansion_panel.listExample}` here.

But unfortunately there is a dartdoc bug that is preventing that at the moment: https://github.com/dart-lang/dartdoc/issues/1945

So, I'd recommend you either duplicate the example here (and maybe customize it for this documentation), or just keep the link as you have it.

Reply:
Probably fine to just leave as is then.